Library/Homebrew/caveats.rb

```diff
@@ -143,12 +143,7 @@ def elisp_caveats
     if keg && keg.elisp_installed?
       <<-EOS.undent
         Emacs Lisp files have been installed to:
-        #{HOMEBREW_PREFIX}/share/emacs/site-lisp/
-
-        Add the following to your init file to have packages installed by
```

Comment:
Removing this since it's not really Homebrew-specific; it was there to help with the transition from putting everything directly in `site-lisp/` to using subdirectories.